inference-engine/tests/functional/plugin/cpu/test_utils/cpu_test_utils.hpp

```diff
@@ -82,6 +88,9 @@ class CPUTestsBase {
     std::vector<cpu_memory_format_t> inFmts, outFmts;
     std::vector<std::string> priority;
     std::string selectedType;
+
+    unsigned getPerCoreCacheSize(int level);
+    int getCacheSize(int level, bool per_core);
```

Comment:
Since they does not change the internal state, it would be better to move them to the public section and make them static class methods.

Reply:
Thanks, I made these methods static